### PR TITLE
HAWQ-191. Remove Analyzer plugin from PXF

### PR DESCRIPTION
**Pull request by shivzone**

### CODE REVIEW

File: pxf/build.gradle [hunk 1]
```diff
@@ -218,7 +218,7 @@ project('pxf-service') {
 
 project('pxf-hdfs') {
     dependencies {
-        compile(project(':pxf-service')) //Yikes, HdfsAnalyzer is directly accessing the bridge
+        compile(project(':pxf-service'))
```
Comment (hornn) [inline pxf/build.gradle]: Can't we replace the dependency with a dependency on pxf-api (like pxf-hbase does)?

**Comment by hornn** [comment]
Removed code looks good.
+0.5 (waiting for the warning message code)

File: pxf/build.gradle (hunk 1 above)
Comment (shivzone) [inline pxf/build.gradle]: There are other compile time dependancies unrelated to the analyzer

File: pxf/pxf-service/src/main/java/org/apache/hawq/pxf/service/rest/InvalidPathResource.java [hunk 2]
```diff
@@ -153,9 +167,39 @@ private String parseVersion(String path) {
     }
 
     /**
+     * Parses the version part from the path.
```
Comment (hornn) [inline pxf/pxf-service/src/main/java/org/apache/hawq/pxf/service/rest/InvalidPathResource.java]: 1. Please update doc - it parses the part after the version part, right? /pxf/version/endpoint
2. could you please refactor parseVersion() to also use PathSegment? 
Actually, both functions can perhaps be consolidated. [e.g. parsePathSegment(path, VERSION), parsePath(path, ENDPOINT)]

Comment (shivzone) [inline pxf/pxf-service/src/main/java/org/apache/hawq/pxf/service/rest/InvalidPathResource.java]: These are private methods used only by wrongPath(). Having an enum and separate function seems to be an overkill for this. will do away with helper functions and use pathSegments directly to extract the version and endpoint.

Comment (hornn) [inline pxf/pxf-service/src/main/java/org/apache/hawq/pxf/service/rest/InvalidPathResource.java]: Sounds good.

File: pxf/build.gradle (hunk 1 above)
Comment (hornn) [inline pxf/build.gradle]: There are 2 other classes using pxf-service. I will open a JIRA to fix it.

**Comment by hornn** [comment]
+1

File: pxf/build.gradle (hunk 1 above)
Comment (shivzone) [inline pxf/build.gradle]: Yes, separate jira makes more sense

File: pxf/pxf-service/src/main/java/org/apache/hawq/pxf/service/rest/InvalidPathResource.java (hunk 2 above)
Comment (shivzone) [inline pxf/pxf-service/src/main/java/org/apache/hawq/pxf/service/rest/InvalidPathResource.java]: Fixed this directly on master

**Pull request closed without merging**